scripts/deploy/deploy-tekton.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+MAX_RETRIES="${MAX_RETRIES:-5}"
+SLEEP_TIME="${SLEEP_TIME:-10}"
+EXIT_CODE=0
+
+TEKTON_NS="${TEKTON_NS:-"tekton-pipelines"}"
+TEKTON_MANIFEST="${TEKTON_MANIFEST:-https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml}"
```

Comment:
nit: it would be good to be able to specify the tekton version

Reply:
Good idea. Added that in 👍